f600233d1b154d507b782f84604bcb12/update/sys_script_include_b29002b097a211106fa8b4b3f153afc4.xml

```diff
@@ -104,311 +134,221 @@ Igce.prototype = {
         });
     },
-    getClinInfo: function (idiqClins) {
-        var idiqFields = ["contract", "idiq_clin", "description", "contract_type", "classification_level"];
-        //put everything into an array
-        var clin_info = getListRecords(idiqClins.toString(), idiqFields, 'x_g_dis_atat_idiq_clin').map(removeSysId);
-        clin_info.forEach(function (item) {
-            if (item.classification_level != null) {
-                var gr = new GlideRecord('x_g_dis_atat_classification_level');
-                gr.get(item.classification_level.toString());
-                var classLevel = gr.getDisplayValue('classification');
-                item.classification_level = classLevel.toUpperCase();
-                // IGCE appends a CLASSFIED to the end of SECRET and TOP SECRET, format string to reflect that difference
-                if (item.classification_level == "SECRET" || item.classification_level == "TOP SECRET") {
-                    item.classification_level += " CLASSIFIED";
-                }
+    getIgceEstimates: function (packageSysId) {
+        var packageId = packageSysId ? packageSysId : this.acquisitionPackage.sys_id;
+        var limit = 50; // arbitrary
+        return new global.GlideQuery(Igce.prototype.IGCE_ESTIMATE_TABLE)
+            .where("acquisition_package", packageId)
+            .select(Igce.prototype.IGCE_ESTIMATE_FIELDS)
+            .toArray(limit);
+    },
+    getIgceInstanceData: function (table, id) {
+        var instance_data = new global.GlideQuery(table).get(id, Igce.prototype.SELECTED_PERIOD_FIELDS).orElse(null);
+        var instance_data_with_periods = Igce.prototype.getInstancePeriodInfo(instance_data);
+        return instance_data_with_periods;
+    },
+    getClassificationLevel: function (classificationId) {
+        var classification_level = new global.GlideQuery(Igce.prototype.CLASSIFICATION_LEVEL_TABLE).get(classificationId, ["classification"]).orElse(null);
+        return classification_level.classification;
+    },
+    determineIdiqClin: function (classification, idiq_clin_type) {
+        if (classification === "U" && idiq_clin_type === "CLOUD") {
+            return "x001/x017 Cloud UNCLASSIFIED"
+        }
+        if (classification === "U" && idiq_clin_type === "CLOUD_SUPPORT") {
+            return "x002/x018 Cloud Support UNCLASSIFIED"
+        }
+        if (classification === "S" && idiq_clin_type === "CLOUD") {
+            return "x003/x019 Cloud SECRET CLASSIFIED"
+        }
+        if (classification === "S" && idiq_clin_type === "CLOUD_SUPPORT") {
+            return "x004/x020 Cloud Support SECRET CLASSIFIED"
+        }
+        if (classification === "TS" && idiq_clin_type === "CLOUD") {
+            return "x005/x021 Cloud TOP SECRET CLASSIFIED"
+        }
+        if (classification === "TS" && idiq_clin_type === "CLOUD_SUPPORT") {
+            return "x006/x022 Cloud Support TOP SECRET CLASSIFIED"
+        }
 
+        // TODO: add option for (x007 Travel)
```

Comment:
Is this for now or for later?

Reply:
Need to confirm where they are saving travel data. If its not in IGCE estimates the table should be refactored on the SNOW side because it doesn't have a DOW task number listed.

Reply:
The estimates for travel data are stored in `Requirements Cost Estimate.travel_estimated_values`.  Sounds like this would be problematic because you'd be missing `dow_task_number`, is that right?

Reply:
I'm inclined to commit what is done and we can work that matter separately.